src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +94,12 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(x -> (x.getSex() == People.Sex.MAN
+                        && (x.getAge() >= fromAge && x.getAge() <= maleToAge))
+                        || (x.getSex() == People.Sex.WOMEN
+                        && (x.getAge() >= fromAge && x.getAge() <= femaleToAge)))
```

Comment:
move `x.getAge() >= fromAge` to the left side of `&&` - it is common for both cases